ocw/lib/EC2.py

```diff
@@ -411,3 +412,69 @@ def cleanup_images(self):
                 else:
                     self.ec2_client(region).deregister_image(ImageId=img.id, DryRun=False)
+
+    def cleanup_k8s_jobs(self,):
+        clusters = dict()
+        for region in self.cluster_regions:
+            response = self.eks_client(region).list_clusters()
+            if 'clusters' in response and len(response['clusters']) > 0:
+                clusters[region] = []
+                for cluster_name in response['clusters']:
+                    cluster_description = self.eks_client(region).describe_cluster(name=cluster_name)
+                    cluster_cert = cluster_description["cluster"]["certificateAuthority"]["data"]
+                    cluster_ep = cluster_description["cluster"]["endpoint"]
+
+                    #https://github.com/cybermaggedon/aws-eks-deployment/blob/master/deploy-k8s
+                    cluster_config = {
+                        "apiVersion": "v1",
+                        "kind": "Config",
+                        "clusters": [
+                            {
+                                "cluster": {
+                                    "server": str(cluster_ep),
+                                    "certificate-authority-data": str(cluster_cert)
+                                },
+                                "name": "kubernetes"
+                            }
+                        ],
+                        "contexts": [
+                            {
+                                "context": {
+                                    "cluster": "kubernetes",
+                                    "user": "aws"
+                                },
+                                "name": "aws"
+                            }
+                        ],
+                        "current-context": "aws",
+                        "preferences": {},
+                        "users": [
+                            {
+                                "name": "aws",
+                                "user": {
+                                    "exec": {
+                                        "apiVersion": "client.authentication.k8s.io/v1alpha1",
+                                        "command": "heptio-authenticator-aws",
+                                        "args": [
+                                            "token", "-i", cluster_name
+                                        ]
+                                    }
+                                }
+                            }
+                        ]
+                    }
+
+                    config_file = f"/tmp/k8s_{region}_{cluster_name}.yaml"
+
+                    # Write in YAML.
+                    config_text=yaml.dump(cluster_config, default_flow_style=False)
+                    open(config_file, "w").write(config_text)
+
+                    self.log_info(f"Created config file {config_file}")
+
+                    resp = subprocess.call(["kubectl", "--kubeconfig=%s" % config_file, "get", "nodes"])
```

Comment:
maybe you look for some python lib for interacting with k8s ?

Reply:
something like this https://github.com/kubernetes-client/python ? But please do better research it is just first URL from google